components/footer.js

```diff
@@ -51,6 +52,11 @@ const FooterText = styled.div`
   margin-top: 0px;
 `
 
+const FooterVersion = styled.div`
+  margin-top: 0px;
+  margin-left: 45px;
+`
+
```

Comment:
If we are only customizing the margins, I think it is better to just use the `Box` component we already use. [This documentation](https://rebass-v2.now.sh/props#margin-and-padding) should help. 

It might also help with the overall consistency if we use the [scale set in the theme](https://github.com/ooni/design-system/blob/master/components/theme/rebassTheme.js#L10-L16) e.g `ml={[1-6]}` instead of a fine tuned value like `45px`

Reply:
That makes total sense. I was trying to keep it consistent with `FooterText` but didn't know about the scale. Thanks for pointing me in the right direction.